### ISSUE TITLE
[FTP] torrent file copy fails

### ISSUE THREAD

**Issue by laur89** [rclone/rclone#6793]
#### What is the problem you are having with rclone?

Copying/moving ~torrent~ larger than 14034 byte file to FTP remote fails.

#### What is your rclone version (output from `rclone version`)

```
rclone v1.61.1
- os/version: debian bookworm/sid (64 bit)
- os/kernel: 6.1.0-3-amd64 (x86_64)
- os/type: linux
- os/arch: amd64
- go/version: go1.19.4
- go/linking: static
- go/tags: none
```

#### Which OS you are using and how many bits (e.g. Windows 7, 64 bit)

Debian testing, 64bit

#### Which cloud storage system are you using? (e.g. Google Drive)

FTPS:

`~/.config/rclone/rclone.conf`:
```
[remote]
type = ftp
host = remote.host.domain
user = laur89
port = 21
pass = <redacted>
tls = false
explicit_tls = true
no_check_certificate = true
```

#### Reproduction
1. save a ~torrent file~ file that's larger than 14034 bytes locally, eg [this ubuntu release](https://releases.ubuntu.com/22.10/ubuntu-22.10-desktop-amd64.iso.torrent) .torrent file.
2. try to copy (or move) this file to your remote FTP host:

```
$ ./rclone copy -vv --dump bodies ../ubuntu-22.10-desktop-amd64.iso.torrent  remote:files/

<7>DEBUG : rclone: Version "v1.61.1" starting with parameters ["./rclone" "copy" "-vv" "--dump" "bodies" "../ubuntu-22.10-desktop-amd64.iso.torrent" "remote:files/"]
<7>DEBUG : rclone: systemd logging support activated
<7>DEBUG : Creating backend with remote "../ubuntu-22.10-desktop-amd64.iso.torrent"
<7>DEBUG : Using config file from "/home/laur/.config/rclone/rclone.conf"
<7>DEBUG : fs cache: adding new entry for parent of "../ubuntu-22.10-desktop-amd64.iso.torrent", "/home/laur/Downloads"
<7>DEBUG : Creating backend with remote "remote:files/"
<7>DEBUG : ftp://remote.host.domain:21/files: Connecting to FTP server
<7>DEBUG : ftp://remote.host.domain:21/files: dial("tcp","remote.host.domain:21")
<7>DEBUG : ftp://remote.host.domain:21/files: > dial: conn=*fshttp.timeoutConn, err=<nil>
<7>DEBUG : FTP Rx: "220 (vsFTPd 3.0.3)"
<7>DEBUG : FTP Tx: "AUTH TLS"
<7>DEBUG : FTP Rx: "234 Proceed with negotiation."
<7>DEBUG : FTP Tx: "USER laur89"
<7>DEBUG : FTP Rx: "331 Please specify the password."
<7>DEBUG : FTP Tx: PASS *****
<7>DEBUG : FTP Rx: "230 Login successful."
<7>DEBUG : FTP Tx: "FEAT"
<7>DEBUG : FTP Rx: "211-Features:"
<7>DEBUG : FTP Rx: " AUTH TLS"
<7>DEBUG : FTP Rx: " UTF8"
<7>DEBUG : FTP Rx: " EPRT"
<7>DEBUG : FTP Rx: " EPSV"
<7>DEBUG : FTP Rx: " MDTM"
<7>DEBUG : FTP Rx: " PASV"
<7>DEBUG : FTP Rx: " PBSZ"
<7>DEBUG : FTP Rx: " PROT"
<7>DEBUG : FTP Rx: " REST STREAM"
<7>DEBUG : FTP Rx: " SIZE"
<7>DEBUG : FTP Rx: " TVFS"
<7>DEBUG : FTP Rx: "211 End"
<7>DEBUG : FTP Tx: "TYPE I"
<7>DEBUG : FTP Rx: "200 Switching to Binary mode."
<7>DEBUG : FTP Tx: "OPTS UTF8 ON"
<7>DEBUG : FTP Rx: "200 Always in UTF8 mode."
<7>DEBUG : FTP Tx: "PBSZ 0"
<7>DEBUG : FTP Rx: "200 PBSZ set to 0."
<7>DEBUG : FTP Tx: "PROT P"
<7>DEBUG : FTP Rx: "200 PROT now Private."
<7>DEBUG : FTP Tx: "EPSV"
<7>DEBUG : FTP Rx: "229 Entering Extended Passive Mode (|||8611|)"
<7>DEBUG : ftp://remote.host.domain:21/files: dial("tcp","46.232.211.100:8611")
<7>DEBUG : ftp://remote.host.domain:21/files: > dial: conn=*tls.Conn, err=<nil>
<7>DEBUG : FTP Tx: "LIST files"
<7>DEBUG : FTP Rx: "150 Here comes the directory listing."
<7>DEBUG : FTP Rx: "drwxr-xr-x    2 1235     1235         4096 Oct 26 13:30 srcdir"
<7>DEBUG : FTP Rx: "drwxr-xr-x    4 1235     1235         4096 Dec 06 01:29 testdir"
<7>DEBUG : FTP Rx: "drwxr-xr-x    2 1235     1235         4096 Feb 28 21:16 watch"
<7>DEBUG : FTP Rx: "drwxr-xr-x    2 1235     1235         4096 Feb 28 21:14 watch-manual"
<7>DEBUG : FTP Rx: "226 Directory send OK."
<7>DEBUG : FTP Tx: "EPSV"
<7>DEBUG : FTP Rx: "229 Entering Extended Passive Mode (|||5925|)"
<7>DEBUG : ftp://remote.host.domain:21/files: dial("tcp","46.232.211.100:5925")
<7>DEBUG : ftp://remote.host.domain:21/files: > dial: conn=*tls.Conn, err=<nil>
<7>DEBUG : FTP Tx: "LIST files"
<7>DEBUG : FTP Rx: "150 Here comes the directory listing."
<7>DEBUG : FTP Rx: "drwxr-xr-x    2 1235     1235         4096 Oct 26 13:30 srcdir"
<7>DEBUG : FTP Rx: "drwxr-xr-x    4 1235     1235         4096 Dec 06 01:29 testdir"
<7>DEBUG : FTP Rx: "drwxr-xr-x    2 1235     1235         4096 Feb 28 21:16 watch"
<7>DEBUG : FTP Rx: "drwxr-xr-x    2 1235     1235         4096 Feb 28 21:14 watch-manual"
<7>DEBUG : FTP Rx: "226 Directory send OK."
<7>DEBUG : ubuntu-22.10-desktop-amd64.iso.torrent: Need to transfer - File not found at Destination
<7>DEBUG : FTP Tx: "EPSV"
<7>DEBUG : FTP Rx: "229 Entering Extended Passive Mode (|||7051|)"
<7>DEBUG : ftp://remote.host.domain:21/files: dial("tcp","46.232.211.100:7051")
<7>DEBUG : ftp://remote.host.domain:21/files: > dial: conn=*tls.Conn, err=<nil>
<7>DEBUG : FTP Tx: "LIST ."
<7>DEBUG : FTP Rx: "150 Here comes the directory listing."
<7>DEBUG : FTP Rx: "-rw-------    1 1235     1235           40 Oct 12  2021 .api-key"
<7>DEBUG : FTP Rx: "drwx------    6 1235     1235         4096 Jan 17  2022 .apps"
<7>DEBUG : FTP Rx: "drwxr-xr-x   12 1235     1235         4096 Feb 28 21:18 files"
<7>DEBUG : FTP Rx: "226 Directory send OK."
<7>DEBUG : FTP Tx: "EPSV"
<7>DEBUG : FTP Rx: "229 Entering Extended Passive Mode (|||9801|)"
<7>DEBUG : ftp://remote.host.domain:21/files: dial("tcp","46.232.211.100:9801")
<7>DEBUG : ftp://remote.host.domain:21/files: > dial: conn=*tls.Conn, err=<nil>
<7>DEBUG : FTP Tx: "STOR files/ubuntu-22.10-desktop-amd64.iso.torrent"
<7>DEBUG : FTP Rx: "150 Ok to send data."
<7>DEBUG : FTP Rx: "426 Failure reading network stream."
<7>DEBUG : FTP Tx: "QUIT"
<7>DEBUG : ftp://remote.host.domain:21/files: Connecting to FTP server
<7>DEBUG : ftp://remote.host.domain:21/files: dial("tcp","remote.host.domain:21")
<7>DEBUG : ftp://remote.host.domain:21/files: > dial: conn=*fshttp.timeoutConn, err=<nil>
<7>DEBUG : FTP Rx: "220 (vsFTPd 3.0.3)"
<7>DEBUG : FTP Tx: "AUTH TLS"
<7>DEBUG : FTP Rx: "234 Proceed with negotiation."
<7>DEBUG : FTP Tx: "USER laur89"
<7>DEBUG : FTP Rx: "331 Please specify the password."
<7>DEBUG : FTP Tx: PASS *****
<7>DEBUG : FTP Rx: "230 Login successful."
<7>DEBUG : FTP Tx: "FEAT"
<7>DEBUG : FTP Rx: "211-Features:"
<7>DEBUG : FTP Rx: " AUTH TLS"
<7>DEBUG : FTP Rx: " UTF8"
<7>DEBUG : FTP Rx: " EPRT"
<7>DEBUG : FTP Rx: " EPSV"
<7>DEBUG : FTP Rx: " MDTM"
<7>DEBUG : FTP Rx: " PASV"
<7>DEBUG : FTP Rx: " PBSZ"
<7>DEBUG : FTP Rx: " PROT"
<7>DEBUG : FTP Rx: " REST STREAM"
<7>DEBUG : FTP Rx: " SIZE"
<7>DEBUG : FTP Rx: " TVFS"
<7>DEBUG : FTP Rx: "211 End"
<7>DEBUG : FTP Tx: "TYPE I"
<7>DEBUG : FTP Rx: "200 Switching to Binary mode."
<7>DEBUG : FTP Tx: "OPTS UTF8 ON"
<7>DEBUG : FTP Rx: "200 Always in UTF8 mode."
<7>DEBUG : FTP Tx: "PBSZ 0"
<7>DEBUG : FTP Rx: "200 PBSZ set to 0."
<7>DEBUG : FTP Tx: "PROT P"
<7>DEBUG : FTP Rx: "200 PROT now Private."
<7>DEBUG : FTP Tx: "EPSV"
<7>DEBUG : FTP Rx: "229 Entering Extended Passive Mode (|||7977|)"
<7>DEBUG : ftp://remote.host.domain:21/files: dial("tcp","46.232.211.100:7977")
<7>DEBUG : ftp://remote.host.domain:21/files: > dial: conn=*tls.Conn, err=<nil>
<7>DEBUG : FTP Tx: "LIST files"
<7>DEBUG : FTP Rx: "150 Here comes the directory listing."
<7>DEBUG : FTP Rx: "drwxr-xr-x    2 1235     1235         4096 Oct 26 13:30 srcdir"
<7>DEBUG : FTP Rx: "drwxr-xr-x    4 1235     1235         4096 Dec 06 01:29 testdir"
<7>DEBUG : FTP Rx: "-rw-r--r--    1 1235     1235        52930 Feb 28 21:18 ubuntu-22.10-desktop-amd64.iso.torrent"
<7>DEBUG : FTP Rx: "drwxr-xr-x    2 1235     1235         4096 Feb 28 21:16 watch"
<7>DEBUG : FTP Rx: "drwxr-xr-x    2 1235     1235         4096 Feb 28 21:14 watch-manual"
<7>DEBUG : FTP Rx: "226 Directory send OK."
<7>DEBUG : FTP Tx: "DELE files/ubuntu-22.10-desktop-amd64.iso.torrent"
<7>DEBUG : FTP Rx: "250 Delete operation successful."
<7>DEBUG : ubuntu-22.10-desktop-amd64.iso.torrent: Removed after failed upload: 1 error occurred:
	* 426 Failure reading network stream.

<3>ERROR : ubuntu-22.10-desktop-amd64.iso.torrent: Failed to copy: update stor: 1 error occurred:
	* 426 Failure reading network stream.

<3>ERROR : Attempt 1/3 failed with 1 errors and: update stor: 1 error occurred:
	* 426 Failure reading network stream.

<7>DEBUG : FTP Tx: "EPSV"
<7>DEBUG : FTP Rx: "229 Entering Extended Passive Mode (|||9849|)"
<7>DEBUG : ftp://remote.host.domain:21/files: dial("tcp","46.232.211.100:9849")
<7>DEBUG : ftp://remote.host.domain:21/files: > dial: conn=*tls.Conn, err=<nil>
<7>DEBUG : FTP Tx: "LIST files"
<7>DEBUG : FTP Rx: "150 Here comes the directory listing."
<7>DEBUG : FTP Rx: "drwxr-xr-x    2 1235     1235         4096 Oct 26 13:30 srcdir"
<7>DEBUG : FTP Rx: "drwxr-xr-x    4 1235     1235         4096 Dec 06 01:29 testdir"
<7>DEBUG : FTP Rx: "drwxr-xr-x    2 1235     1235         4096 Feb 28 21:16 watch"
<7>DEBUG : FTP Rx: "drwxr-xr-x    2 1235     1235         4096 Feb 28 21:14 watch-manual"
<7>DEBUG : FTP Rx: "226 Directory send OK."
<7>DEBUG : ubuntu-22.10-desktop-amd64.iso.torrent: Need to transfer - File not found at Destination
<7>DEBUG : FTP Tx: "EPSV"
<7>DEBUG : FTP Rx: "229 Entering Extended Passive Mode (|||9307|)"
<7>DEBUG : ftp://remote.host.domain:21/files: dial("tcp","46.232.211.100:9307")
<7>DEBUG : ftp://remote.host.domain:21/files: > dial: conn=*tls.Conn, err=<nil>
<7>DEBUG : FTP Tx: "LIST ."
<7>DEBUG : FTP Rx: "150 Here comes the directory listing."
<7>DEBUG : FTP Rx: "-rw-------    1 1235     1235           40 Oct 12  2021 .api-key"
<7>DEBUG : FTP Rx: "drwx------    6 1235     1235         4096 Jan 17  2022 .apps"
<7>DEBUG : FTP Rx: "drwxr-xr-x   12 1235     1235         4096 Feb 28 21:18 files"
<7>DEBUG : FTP Rx: "226 Directory send OK."
<7>DEBUG : FTP Tx: "EPSV"
<7>DEBUG : FTP Rx: "229 Entering Extended Passive Mode (|||7266|)"
<7>DEBUG : ftp://remote.host.domain:21/files: dial("tcp","46.232.211.100:7266")
<7>DEBUG : ftp://remote.host.domain:21/files: > dial: conn=*tls.Conn, err=<nil>
<7>DEBUG : FTP Tx: "STOR files/ubuntu-22.10-desktop-amd64.iso.torrent"
<7>DEBUG : FTP Rx: "150 Ok to send data."
<7>DEBUG : FTP Rx: "426 Failure reading network stream."
<7>DEBUG : FTP Tx: "QUIT"
<7>DEBUG : ftp://remote.host.domain:21/files: Connecting to FTP server
<7>DEBUG : ftp://remote.host.domain:21/files: dial("tcp","remote.host.domain:21")
<7>DEBUG : ftp://remote.host.domain:21/files: > dial: conn=*fshttp.timeoutConn, err=<nil>
<7>DEBUG : FTP Rx: "220 (vsFTPd 3.0.3)"
<7>DEBUG : FTP Tx: "AUTH TLS"
<7>DEBUG : FTP Rx: "234 Proceed with negotiation."
<7>DEBUG : FTP Tx: "USER laur89"
<7>DEBUG : FTP Rx: "331 Please specify the password."
<7>DEBUG : FTP Tx: PASS *****
<7>DEBUG : FTP Rx: "230 Login successful."
<7>DEBUG : FTP Tx: "FEAT"
<7>DEBUG : FTP Rx: "211-Features:"
<7>DEBUG : FTP Rx: " AUTH TLS"
<7>DEBUG : FTP Rx: " UTF8"
<7>DEBUG : FTP Rx: " EPRT"
<7>DEBUG : FTP Rx: " EPSV"
<7>DEBUG : FTP Rx: " MDTM"
<7>DEBUG : FTP Rx: " PASV"
<7>DEBUG : FTP Rx: " PBSZ"
<7>DEBUG : FTP Rx: " PROT"
<7>DEBUG : FTP Rx: " REST STREAM"
<7>DEBUG : FTP Rx: " SIZE"
<7>DEBUG : FTP Rx: " TVFS"
<7>DEBUG : FTP Rx: "211 End"
<7>DEBUG : FTP Tx: "TYPE I"
<7>DEBUG : FTP Rx: "200 Switching to Binary mode."
<7>DEBUG : FTP Tx: "OPTS UTF8 ON"
<7>DEBUG : FTP Rx: "200 Always in UTF8 mode."
<7>DEBUG : FTP Tx: "PBSZ 0"
<7>DEBUG : FTP Rx: "200 PBSZ set to 0."
<7>DEBUG : FTP Tx: "PROT P"
<7>DEBUG : FTP Rx: "200 PROT now Private."
<7>DEBUG : FTP Tx: "EPSV"
<7>DEBUG : FTP Rx: "229 Entering Extended Passive Mode (|||5325|)"
<7>DEBUG : ftp://remote.host.domain:21/files: dial("tcp","46.232.211.100:5325")
<7>DEBUG : ftp://remote.host.domain:21/files: > dial: conn=*tls.Conn, err=<nil>
<7>DEBUG : FTP Tx: "LIST files"
<7>DEBUG : FTP Rx: "150 Here comes the directory listing."
<7>DEBUG : FTP Rx: "drwxr-xr-x    2 1235     1235         4096 Oct 26 13:30 srcdir"
<7>DEBUG : FTP Rx: "drwxr-xr-x    4 1235     1235         4096 Dec 06 01:29 testdir"
<7>DEBUG : FTP Rx: "-rw-r--r--    1 1235     1235       114688 Feb 28 21:18 ubuntu-22.10-desktop-amd64.iso.torrent"
<7>DEBUG : FTP Rx: "drwxr-xr-x    2 1235     1235         4096 Feb 28 21:16 watch"
<7>DEBUG : FTP Rx: "drwxr-xr-x    2 1235     1235         4096 Feb 28 21:14 watch-manual"
<7>DEBUG : FTP Rx: "226 Directory send OK."
<7>DEBUG : FTP Tx: "DELE files/ubuntu-22.10-desktop-amd64.iso.torrent"
<7>DEBUG : FTP Rx: "250 Delete operation successful."
<7>DEBUG : ubuntu-22.10-desktop-amd64.iso.torrent: Removed after failed upload: 1 error occurred:
	* 426 Failure reading network stream.

<3>ERROR : ubuntu-22.10-desktop-amd64.iso.torrent: Failed to copy: update stor: 1 error occurred:
	* 426 Failure reading network stream.

<3>ERROR : Attempt 2/3 failed with 1 errors and: update stor: 1 error occurred:
	* 426 Failure reading network stream.

<7>DEBUG : FTP Tx: "EPSV"
<7>DEBUG : FTP Rx: "229 Entering Extended Passive Mode (|||6650|)"
<7>DEBUG : ftp://remote.host.domain:21/files: dial("tcp","46.232.211.100:6650")
<7>DEBUG : ftp://remote.host.domain:21/files: > dial: conn=*tls.Conn, err=<nil>
<7>DEBUG : FTP Tx: "LIST files"
<7>DEBUG : FTP Rx: "150 Here comes the directory listing."
<7>DEBUG : FTP Rx: "drwxr-xr-x    2 1235     1235         4096 Oct 26 13:30 srcdir"
<7>DEBUG : FTP Rx: "drwxr-xr-x    4 1235     1235         4096 Dec 06 01:29 testdir"
<7>DEBUG : FTP Rx: "drwxr-xr-x    2 1235     1235         4096 Feb 28 21:16 watch"
<7>DEBUG : FTP Rx: "drwxr-xr-x    2 1235     1235         4096 Feb 28 21:14 watch-manual"
<7>DEBUG : FTP Rx: "226 Directory send OK."
<7>DEBUG : ubuntu-22.10-desktop-amd64.iso.torrent: Need to transfer - File not found at Destination
<7>DEBUG : FTP Tx: "EPSV"
<7>DEBUG : FTP Rx: "229 Entering Extended Passive Mode (|||6564|)"
<7>DEBUG : ftp://remote.host.domain:21/files: dial("tcp","46.232.211.100:6564")
<7>DEBUG : ftp://remote.host.domain:21/files: > dial: conn=*tls.Conn, err=<nil>
<7>DEBUG : FTP Tx: "LIST ."
<7>DEBUG : FTP Rx: "150 Here comes the directory listing."
<7>DEBUG : FTP Rx: "-rw-------    1 1235     1235           40 Oct 12  2021 .api-key"
<7>DEBUG : FTP Rx: "drwx------    6 1235     1235         4096 Jan 17  2022 .apps"
<7>DEBUG : FTP Rx: "drwxr-xr-x   12 1235     1235         4096 Feb 28 21:18 files"
<7>DEBUG : FTP Rx: "226 Directory send OK."
<7>DEBUG : FTP Tx: "EPSV"
<7>DEBUG : FTP Rx: "229 Entering Extended Passive Mode (|||9799|)"
<7>DEBUG : ftp://remote.host.domain:21/files: dial("tcp","46.232.211.100:9799")
<7>DEBUG : ftp://remote.host.domain:21/files: > dial: conn=*tls.Conn, err=<nil>
<7>DEBUG : FTP Tx: "STOR files/ubuntu-22.10-desktop-amd64.iso.torrent"
<7>DEBUG : FTP Rx: "150 Ok to send data."
<7>DEBUG : FTP Rx: "426 Failure reading network stream."
<7>DEBUG : FTP Tx: "QUIT"
<7>DEBUG : ftp://remote.host.domain:21/files: Connecting to FTP server
<7>DEBUG : ftp://remote.host.domain:21/files: dial("tcp","remote.host.domain:21")
<7>DEBUG : ftp://remote.host.domain:21/files: > dial: conn=*fshttp.timeoutConn, err=<nil>
<7>DEBUG : FTP Rx: "220 (vsFTPd 3.0.3)"
<7>DEBUG : FTP Tx: "AUTH TLS"
<7>DEBUG : FTP Rx: "234 Proceed with negotiation."
<7>DEBUG : FTP Tx: "USER laur89"
<7>DEBUG : FTP Rx: "331 Please specify the password."
<7>DEBUG : FTP Tx: PASS *****
<7>DEBUG : FTP Rx: "230 Login successful."
<7>DEBUG : FTP Tx: "FEAT"
<7>DEBUG : FTP Rx: "211-Features:"
<7>DEBUG : FTP Rx: " AUTH TLS"
<7>DEBUG : FTP Rx: " UTF8"
<7>DEBUG : FTP Rx: " EPRT"
<7>DEBUG : FTP Rx: " EPSV"
<7>DEBUG : FTP Rx: " MDTM"
<7>DEBUG : FTP Rx: " PASV"
<7>DEBUG : FTP Rx: " PBSZ"
<7>DEBUG : FTP Rx: " PROT"
<7>DEBUG : FTP Rx: " REST STREAM"
<7>DEBUG : FTP Rx: " SIZE"
<7>DEBUG : FTP Rx: " TVFS"
<7>DEBUG : FTP Rx: "211 End"
<7>DEBUG : FTP Tx: "TYPE I"
<7>DEBUG : FTP Rx: "200 Switching to Binary mode."
<7>DEBUG : FTP Tx: "OPTS UTF8 ON"
<7>DEBUG : FTP Rx: "200 Always in UTF8 mode."
<7>DEBUG : FTP Tx: "PBSZ 0"
<7>DEBUG : FTP Rx: "200 PBSZ set to 0."
<7>DEBUG : FTP Tx: "PROT P"
<7>DEBUG : FTP Rx: "200 PROT now Private."
<7>DEBUG : FTP Tx: "EPSV"
<7>DEBUG : FTP Rx: "229 Entering Extended Passive Mode (|||5770|)"
<7>DEBUG : ftp://remote.host.domain:21/files: dial("tcp","46.232.211.100:5770")
<7>DEBUG : ftp://remote.host.domain:21/files: > dial: conn=*tls.Conn, err=<nil>
<7>DEBUG : FTP Tx: "LIST files"
<7>DEBUG : FTP Rx: "150 Here comes the directory listing."
<7>DEBUG : FTP Rx: "drwxr-xr-x    2 1235     1235         4096 Oct 26 13:30 srcdir"
<7>DEBUG : FTP Rx: "drwxr-xr-x    4 1235     1235         4096 Dec 06 01:29 testdir"
<7>DEBUG : FTP Rx: "-rw-r--r--    1 1235     1235       114688 Feb 28 21:18 ubuntu-22.10-desktop-amd64.iso.torrent"
<7>DEBUG : FTP Rx: "drwxr-xr-x    2 1235     1235         4096 Feb 28 21:16 watch"
<7>DEBUG : FTP Rx: "drwxr-xr-x    2 1235     1235         4096 Feb 28 21:14 watch-manual"
<7>DEBUG : FTP Rx: "226 Directory send OK."
<7>DEBUG : FTP Tx: "DELE files/ubuntu-22.10-desktop-amd64.iso.torrent"
<7>DEBUG : FTP Rx: "250 Delete operation successful."
<7>DEBUG : ubuntu-22.10-desktop-amd64.iso.torrent: Removed after failed upload: 1 error occurred:
	* 426 Failure reading network stream.

<3>ERROR : ubuntu-22.10-desktop-amd64.iso.torrent: Failed to copy: update stor: 1 error occurred:
	* 426 Failure reading network stream.

<3>ERROR : Attempt 3/3 failed with 1 errors and: update stor: 1 error occurred:
	* 426 Failure reading network stream.

<6>INFO  : 
Transferred:   	  911.238 KiB / 911.238 KiB, 100%, 82.840 KiB/s, ETA 0s
Errors:                 1 (retrying may help)
Elapsed time:        15.3s

<7>DEBUG : 3 go routines active
<7>DEBUG : ftp://remote.host.domain:21/files: closing 1 unused connections
<7>DEBUG : FTP Tx: "QUIT"
Failed to copy: update stor: 1 error occurred:
	* 426 Failure reading network stream.
```

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by ncw** [comment]
Could the FTP host provider be filtering out the `.torrent` files? Try renaming it to `.bin` and uploading it.

**Comment by laur89** [comment]
Nope that's not it. Also tried compressing it into `tar.bz2`, no dice.

**Comment by ncw** [comment]
Can you see which files you can transfer and which you can't and try to make a pattern?

**Comment by laur89** [comment]
> Can you see which files you can transfer and which you can't and try to make a pattern?

So far the pattern seems to be any torrent file.

**Comment by ncw** [comment]
How big are these files? Does it have to have `.torrent` in the name? Do other binary or text files of about the same size transfer properly? What if you rename them to be .torrent?

**Comment by laur89** [comment]
Did some more digging. Looks like it's a function of file size, not type.
Given this [namecheck](https://manpages.debian.org/testing/devscripts/namecheck.1.en.html) perl script, and making duplicates of it while increasing the filesize: 

```bash
-rwxr-xr-x 1 laur laur 5.2K Feb 19 00:56 namecheck
-rwxr-xr-x 1 laur laur  11K Mar  3 13:31 namecheck2  <-- contents duplicated, ie namecheck*2
-rwxr-xr-x 1 laur laur  16K Mar  3 13:31 namecheck3  <-- triple, ie namecheck*3
```

Then only namecheck & namecheck2 transfer fine.

This time I also noticed new error that has missed me until now:
```
<3>ERROR : ftp://remote.host.domain:21/files: SetModTime is not supported
```

I'm guessing this is an issue with the server itself?

**Comment by laur89** [comment]
Found the threshold - 14034 byte file transfers fine, add one more byte and it starts failing. _Sometimes_ the larger transfer retry succeeds, but seldom.

**Comment by ncw** [comment]
Good detective work.

A size based thing makes much more sense than the name. So at around 14k files stop transferring.

I see from the logs the FTP server is `vsFTPd 3.0.3`

I did some googling for the error `426 Failure reading network stream` and it seems to be caused by TLS connections not terminating properly.

This could be an rclone bug or it could be an FTP server bug.

Can you get the logs of the FTP server?

Can you try the transfer without TLS?

**Comment by laur89** [comment]
> Can you get the logs of the FTP server?

Opened a ticket with the provider, will have to come back.
 
> Can you try the transfer without TLS?

This provider is requires its users to connect using explicit TLS.

**Comment by ncw** [comment]
Thank you for the debug logs.

> Tue Mar 14 02:59:56 2023 [pid 62425] [ultradocs] FTP response: Client "::ffff:46.232.211.80", "426 Failure reading network stream."
> Tue Mar 14 02:59:56 2023 [pid 62425] [ultradocs] FAIL UPLOAD: Client "::ffff:46.232.211.80", "/test.data", 11860 bytes, 2992.77Kbyte/sec

Unfortunately the server log says essentially the same thing that rclone reports and doesn't give any more detail.

Is it possible to get that log with more debugging enabled?

Did you try [the latest beta](https://beta.rclone.org/) - that is worth a go since we updated the FTP library.

This seems very similar to this problem https://bbs.archlinux.org/viewtopic.php?id=246845 which is lftp with vsftpd showing the same error on small transfers which tends to indicate that rclone and vsftpd are having a disagreement as to whether the socket is shut or not.

Alas vsftpd does not appear to have a bug tracker to see if anyone else has reported problems.

**Comment by laur89** [comment]
> Unfortunately the server log says essentially the same thing that rclone reports and doesn't give any more detail.
Suspected that, log didn't look too descriptive.

> Is it possible to get that log with more debugging enabled?
Will ask.

> Did you try [the latest beta](https://beta.rclone.org/) - that is worth a go since we updated the FTP library.
Both the 62 beta and the now-released v62 have a regression - even `lsf` fails with an error:
```
./rclone lsf --max-depth 2  remote:files/target
Failed to create file system for "remote:files/target": 2 errors occurred:
	* remote error: tls: illegal parameter
	* unexpected EOF
```

**Comment by ncw** [comment]
Can you check out this forum thread: https://forum.rclone.org/t/vsftpd-vs-rclone-part-2/36774/ - this seems to have a very similar problem.

Though I have no idea why the `tls: illegal parameter` error is happening. That is probably due to a go update.

I really need to be able to replicate this myself - any chance you could email me some login details to nick@craig-wood.com (put a link to this issue in the email so I have some context) - thanks.

**Comment by ncw** [comment]
I've merged a potential fix to the 426 errors

[v1.63.0-beta.6812.0d8350d95](https://beta.rclone.org/v1.63.0-beta.6812.0d8350d95/) on branch [master](https://github.com/rclone/rclone/tree/master) (uploaded in 15-30 mins)

This won't fix the `tls` errors though - you might want to try this flag for that.

      --ftp-disable-tls13        Disable TLS 1.3 (workaround for FTP servers with buggy TLS)

**Comment by laur89** [comment]
v63 beta works, files copy just fine!
And `--ftp-disable-tls13` option fixes the "tls: illegal param" error -- with for v62.

Do you still need my FTP node login deets?

**Comment by ncw** [comment]
It sounds like we are all done :-) For posterity the fix was 0d8350d95d9bafc4f92746cb4ee43fc4ff662e83

I'll be releasing v1.62.2 pretty soon with that fix in.

Note that you can put `disable_tls13 = true` in the config for the ftp remote which you may find more convenient.

I'll close this now.

Thank you

**Comment by joshenders** [comment]
`disable_tls13 = true` fixed the issue I was having with `vsftpd 3.0.3-13+b2`. Not ideal as TLS 1.3 is quite nice but a good work-around. Thanks!
